### DISCUSSION TITLE
如何用 Tree 实现搜索时隐藏无关节点，类似 TreeSelect ？

### DISCUSSION THREAD

**Discussion by Astra-RX** [ant-design/ant-design#32083] [Q&A] [answered]
好像没有现成的功能，自带的只有高亮，没有更进一步的样式控制，也不像 TreeSelect 那么开箱即用。建议增强一下功能，迁移部分 TreeSelect 的功能回来。

**Comment by afc163** [comment, marked as answer]
确实要支持，可以跟踪 https://github.com/ant-design/ant-design/issues/27197#issuecomment-711488311
